dev-support/ranger-docker/README.md

```diff
@@ -30,9 +30,13 @@ Docker files in this folder create docker images and run them to build Apache Ra
 
 3. Set ```dev-support/ranger-docker``` as your working directory.
 
-4. Execute following command to download necessary archives to setup Ranger/HDFS/Hive/HBase/Kafka/Knox services:
+4. Execute following command to download necessary archives to setup Ranger/HDFS/Hive/HBase/Kafka/Knox/Ozone services:
    ~~~
-   chmod +x download-archives.sh && ./download-archives.sh
+   chmod +x download-archives.sh
+   source download-archives.sh
+   
+   # use a subset of the below to download specific services
+   download_hadoop; download_hbase; download_hive; download_kafka; download_knox; download_ozone
```

Comment:
above `source` command exports methods defined in download-archives.sh - `download_hadoop()`, `download_hbase()`, `download_..()`. These methods don't need to be available in the shell. Instead of this approach, consider updating `download-archives.sh` to take the list of components to download as arguments, as shown below:

`./download-archives.sh hadoop hbase kafka`